tests/unit/libbpf_test.cpp

```diff
@@ -633,6 +613,8 @@ TEST_CASE("libbpf program attach", "[libbpf]")
 }
 #endif
 
+#if 0
+// TODO(#2974): Once the libbpf APIs are implemented in the xdp-for-windows repo, these tests should be migrated.
```

Comment:
I think there should be _negative_ tests in this repo.  That is, verify the relevant libbpf apis gracefully fail (not crash) when xdp is not installed.

Reply:
I have added a set of negative tests which invoke the APIs.